Sources/core/objects/Ammunationboxes.cpp

```diff
@@ -0,0 +1,28 @@
+#include "Ammunationboxes.h"
+
+AmmunitionBox::AmmunitionBox(size_t id) : Item(id)
+{
+    SetSprite("icons/ammo.dmi");
+    SetState("357");
+
+    name = "Revolver Ammunition",
+	
+    bullets_ = 50;
+}
+
+bool AmmunitionBox::CheckBullets()
+{
+    if(bullets_>0)
+    {
```

Comment:
It is needed to place a space between `if` and open bracket. `if (something)`
It is neeed to place a space around operators. `if (bullets_ > 0)`